src/badges.js

```diff
@@ -18,6 +18,72 @@ function removeElementsByClass (className) {
 
 const DOI_REGEX = /(10.\d{4,9}\/[-._;()/:A-Z0-9]+)/ig
 
+const commonOverlayStyle = `
+<style>
+.scite-badge {
+  display: block !important;
+  margin: 0.25rem 0 !important;
+  width: max-content !important;
+  z-index: 9999999999;
+}
+</style>
+`
+
+const commonStyle = `
+    <style>
+    .scite-badge {
+      display: block !important;
+      margin: 0.25rem 0 !important;
+      width: max-content !important;
+    }
+    </style>
+`
+
+const commonMinStyle = `
+    <style>
+    .scite-badge {
+      display: block;
+      width: min-content;
+      margin: 0.25rem 0;
+    }
+    </style>
+`
+
+const xivStyle = `
+    <style>
+    .scite-badge {
+      display: block !important;
+      margin: 0.25rem 0 !important;
+      width: max-content !important;
+    }
+    .scite-badge div {
+      margin: revert !important;
+      padding: 0 0.125rem !important
+    }
+    </style>
+`
+
+const leftMarginStyle = `
+<style>
+.scite-badge {
+  display: block;
+  width: min-content;
+  margin: 0.5rem 0;
+  margin-left: auto;
+}
+</style>
+`
+
+const largeMarginMinStyle = `
+<style>
+.scite-badge {
+  display: block;
+  width: min-content;
+  margin: 0.5rem 0;
+}
+</style>
+`
+
```

Comment:
Is it possible to just import these as CSS modules styles like the main styles?

Reply:
hmm good point I will look into it

Reply:
Not without a rethink on how to change badge styles... since css modules causes a suffix to be appended and I do want to target that specific class. I suppose I could add the classname to the badge itself but I want to think about it a bit more in conjunction with https://github.com/scitedotai/scite/issues/2075 since not sure what I am going to do there yet.